include/deal.II/base/conditional_ostream.h

```diff
@@ -129,12 +134,12 @@ class ConditionalOStream
   const ConditionalOStream &
   operator<< (std::ostream& (*p) (std::ostream &)) const;
 
-private:
   /**
-   * Reference to the stream we want to write to.
+   * The stream we want to write to.
    */
-  std::ostream  &output_stream;
+  std::ostream  output_stream;
```

Comment:
Have you tried to get access to the underlying stream object with <code>ConditionalOstream::get_stream()</code>? Did this fail?